lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+class Menu
+  def initialize
+    hello
+    dishes
+  end
+
+  def dishes
+    @dishes =
+      { name: 'Burger', price: 10.95 },
+      { name: 'Pizza', price: 14.00 },
+      { name: 'Salad', price: 7.60 },
+      { name: 'fries', price: 2.90 }
+  end
+
+  def hello
+    puts 'Welcome to The Greasy Diner'
+    puts 'please make your selection:'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not write to stdout. Use Rails' logger if you want to log.